lib/components/CheckboxButtons/CheckboxButtons.css.ts

```diff
@@ -0,0 +1,125 @@
+import { style } from '@vanilla-extract/css';
+import { recipe } from '@vanilla-extract/recipes';
+
+import { tokens } from 'lib/themes/base/tokens';
+
+import { odStyle, type ODStyle } from '../../styles/sprinkles.css';
+
+const border: ODStyle = {
+	borderColor: 'gray',
+	borderStyle: 'solid',
+	borderWidth: '1',
+};
+
+const focusOutline: ODStyle = {
+	outlineColor: 'link',
+	outlineStyle: 'solid',
+	outlineOffset: 'md',
+	outlineWidth: { initial: 'none', focusVisible: 'default' },
+};
+
+const buttonBorderRadius = tokens.border.radius['2'];
+
+export const checkboxButton = recipe({
+	base: [
+		odStyle({
+			background: {
+				initial: 'white',
+				hover: 'gray200',
+			},
+			...border,
+			cursor: { hover: 'pointer' },
+			display: 'flex',
+			gap: '2',
+			paddingX: '4',
+			paddingY: '3',
+			width: '100%',
+		}),
+		style({
+			selectors: {
+				['&+&']: {
+					borderTopStyle: 'none',
+				},
+				['&:first-child']: {
+					borderTopLeftRadius: buttonBorderRadius,
+					borderTopRightRadius: buttonBorderRadius,
+				},
+				['&:last-child']: {
+					borderBottomLeftRadius: buttonBorderRadius,
+					borderBottomRightRadius: buttonBorderRadius,
+				},
+			},
+		}),
+	],
+	variants: {
+		disabled: {
```

Comment:
The pattern we have been using so far for the disabled interactions is to also set the cursor to `not-allowed`

Reply:
The disabled state has not been designed so I need to work through it with Blake. This will be refactored after the next checkbox with icon iteration because I've learned how to use more features of `sprinkles` 😃